misc/changelog-generator/instructions.md

```diff
@@ -0,0 +1,30 @@
+
+### Creating Pull Requests for Inclusion in NEWS.md
+
+If a PR is significant enough to warrant a mention in the next release notes update,
+its name should begin with a prefix. 
+There are three options depending on the PR's
+purpose.
+
+  * `breaking-change: ` if the change will make the next tag non-backward-compatible
```

Comment:
should we go through our closed PRs from after the 0.6.85 release and update the prefix?